src/Common/src/CoreLib/System/Text/TrimType.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.Text
+{
+    /// <summary>
+    /// Specifies which portions of the string should be trimmed in a trimming operation.
+    /// </summary>
+    [Flags]
+    internal enum TrimType
+    {
+        /// <summary>
+        /// Trim from the beginning of the string.
+        /// </summary>
+        Head = 1 << 0,
```

Comment:
@GrabYourPitchforks question out of curiosity, is this better than declaring flags in hex (i.e: `0x01`) other than just for readability purposes of knowing what bit is on?

Reply:
For readability in `[Flags]`-style enums. Then the values immediately before the comma read 0, 1, 2, 3, ... all the way down the enum, and it's easier to find mistakes or holes.

Reply:
Cool, thanks 😄